From b6416a5ae196649be034a82a2a28e74135022bd7 Mon Sep 17 00:00:00 2001
From: Valentin David <me@valentindavid.com>
Date: Thu, 10 Sep 2026 21:41:41 +0200
Subject: [PATCH 4/6] linux: Enable zswap by default

GNOME OS is switch to using zswap. And it makes it more
convenient to have a sensible default configuration: enabled
an using zstd.
---
 files/linux/fdsdk-config.sh | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/files/linux/fdsdk-config.sh b/files/linux/fdsdk-config.sh
index 4730bf6f7..bcef6a3e8 100644
--- a/files/linux/fdsdk-config.sh
+++ b/files/linux/fdsdk-config.sh
@@ -2386,6 +2386,10 @@ module BLK_DEV_RAM
 
 # ZSWAP
 enable ZSWAP
+enable ZSWAP_COMPRESSOR_DEFAULT_ZSTD
+remove ZSWAP_COMPRESSOR_DEFAULT_LZO
+enable ZSWAP_DEFAULT_ON
+enable ZSWAP_SHRINKER_DEFAULT_ON
 
 # ZRAM
 module ZRAM
-- 
2.55.0

